contracts/crowdloan/Crowdloan.sol

```diff
@@ -25,6 +25,8 @@ contract Crowdloan is Initializable, ICrowdloan, ReentrancyGuard {
     ITermsContract public termsContract;
     IRepaymentManager public repaymentManager;
 
+    uint256 totalCrowdfunded;
```

Comment:
what is the purpose of `totalCrowdfunded`,
why do we have to update it on each fund, and not just pick the value once loan is changed from `funding` phase to `repayment` phase?

Reply:
That makes sense - let me update this